.travis.yml

```diff
@@ -11,5 +11,16 @@ cache:
   directories:
     - node_modules
 
-script:
-    - yarn run jest --projects jest.*.config.js --maxWorkers=4
+script: yarn run jest --projects jest.*.config.js --maxWorkers=4
+
+before_deploy: yarn build
+
+deploy:
+  provider: npm
+  email: npmjs@commercetools.com
+  api_key:
+    secure: KSowahYa4Q4vtLEIaetRTdAZ9cdHwD4c5mnpK0DFi21L0BNUjBzcaocR5Hr3PV62fzdFDHViZ36I3vWojznCmlq1POODHHrFoNycWk+wpI3NJQ887zNbsDIhpDqwN4HPv0TuxfS1r5WiApE6ynm0g+f7DL42+1FAnIIDyKsTNHlimRWuQVk7psinvNs4AyjEyRbvHCEDyRHBXbphX48J1ZUu/uZ9xNIsiWG/1RH0WH1lcT6VHctpy9ZXvmLKw0UilMkbwrNen5OHLQVxp4SrT/4+rxh/msdGwkiH00hi6C0DZizkXxZ+8SGoi//r3l3qMMdhjahBcIp/CEqYu+g59CXIRU1mTEAGf1el9K2dG/cdt9U9D/pQJzlVKgDNsB8XN70wkpoIgdqy2GsEHvth589hsxOI7HlP4NyO36Cv0kIktJDQOwhNd7UQ11p4D5aK8laHcYSD1SUF5sTYpFb8UB8mruWVGf7/siQQsnv9Q+CVkjJKwxvVJlBDr52jQ84+w3FaCoTeVSwOAxTEuK5x9Q7V2KUSxWVDd4MgNWoQkijFEqfe2QbzCW4qpsI3qXSwNKaMYKclodSD93KaTX8thZPy4MCbBKLdX3z8a/ibCbZa4Z83yo1ASFLbFYDimQFjSlQdeV+sia9A36iZSOvSnHqPhj6vhsaFvzgSA06YIjk=
```

Comment:
(noob) question: is it okay to have this committed? is this an encrypted value that travis will decrypt?

Reply:
Yes it's safe: `travis encrypt SECRET`

https://docs.travis-ci.com/user/encryption-keys/

Reply:
@emmenko thanks!